example/ddex/test/meta-claims-service.spec.js

```diff
@@ -0,0 +1,38 @@
+const micro = require('micro')
+const listen = require('test-listen')
+const request = require('request-promise')
+
+const metaClaimsService = require('../')
+const claimData = require('./fixtures/claim.json')
+const verifiedClaimData = require('./fixtures/verified-claim.json')
+
+describe('META Claims Service', () => {
+  let service
+
+  beforeAll(async () => {
+    service = micro(metaClaimsService)
+  })
+
+  afterAll(() => {
+    service.close()
+  })
+
+  it('Should return a META Claim object', async () => {
+    const uri = await listen(service)
+
+    const body = await request({
+      body: claimData,
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      json: true,
+      method: 'POST',
+      uri,
+    })
+
+    const actual = body
+    const expected = verifiedClaimData
+
+    expect(actual).toEqual(expected)
+  })
+})
```

Comment:
There should be a test for an unsuccessful claim; ie. DPID not found in the "DB".

Reply:
Agreed. Will raise an issue.